src/webgl-preview.ts

```diff
@@ -622,15 +679,26 @@ export class WebGLPreview {
       curves.push(new CatmullRomCurve3(curvePoints, false, 'catmullrom', 0));
     }
 
+    console.log('# curves', curves.length);
     curves.forEach((curve) => {
       const material = new MeshLambertMaterial({ color: color });
       this.disposables.push(material);
       const segments = Math.ceil(curve.getLength() * 2);
+      console.log(' # segments', segments);
       const geometry = new TubeGeometry(curve, segments, 0.3, 4, false);
       this.disposables.push(geometry);
-      const lineSegments = new Mesh(geometry, material);
+      // const lineSegments = new Mesh(geometry, material);
+
+      const wireframe = new WireframeGeometry(geometry);
+
+      const line = new LineSegments(wireframe);
+      // line.material.depthTest = false;
+      // line.material.opacity = 0.25;
+      // line.material.transparent = true;
+
+      this.group?.add(line);
 
-      this.group?.add(lineSegments);
+      // this.group?.add(lineSegments);
```

Comment:
Today I learned that there is a better way to render wireframes when debugging a mesh!

The [materials have it built in](https://threejs.org/docs/index.html#api/en/materials/MeshLambertMaterial.wireframe)!!

Try 
```Typescript
const material = new MeshLambertMaterial({ color: color, wireframe: true });

```